specification/softwareplan/resource-manager/Microsoft.SoftwarePlan/stable/2021-08-01/softwareplan.json

```diff
@@ -367,9 +367,46 @@
         "message": {
           "type": "string",
           "description": "A user readable error message. Localized based on x-ms-effective-locale header in the request"
+        },
```

Comment:
FYI, for Error definition, you can re-use common definition in common-types/resource-management/v2/types.json
Let us know if you have special reason not to use it.

Reply:
Thanks, @akning-ms, I've updated it to use the common error definition.